README.md

```diff
@@ -91,6 +91,12 @@ Tangle-accelerator is built and launched through Bazel, it also requires Redis t
 * openssl-dev (required by mosquitto)
 * uuid-dev
 
+In addition, the user should ensure clock synchronization on the machine which tangle-accelerator is running on. This command can help adjusting the time server.
```

Comment:
Change "which tangle-accelerator is running on" to "on which tangle-accelerator is running"

Reply:
Change "help adjusting" to "help the user adjust"
See https://dictionary.cambridge.org/dictionary/english/help

Reply:
It is not meant to adjust time server. In fact, it sets date and time via NTP.